terraform/modules/app-ecs-services/templates/alertmanager.tpl

```diff
@@ -63,7 +63,13 @@ route:
       receiver: "dev-null"
     - match_re:
```

Comment:
I think (but im not sure) that you might need to add a dev-null receiver for the ones that don't match... like:

```
      - match_re:
          namespace: verify-doc-checking-.*
        receiver: dev-null
        routes:
        - receiver: dcs-slack
          match: 
            type: pipeline-alert
        - receiver: dcs-slack
          match:
            namespace: verify-doc-checking-prod
```

which I think will:

* route to dcs-slack if type:pipeline-alert AND namespace:verify-doc-checking-*
* route to dcs-slack if namespace:verify-doc-checking-prod AND namespace:verify-doc-checking-*
* else route to dev-null 

Maybe it bubbles up to the next receiver up, but I'm not 100% sure